packages/react-combobox/src/components/Option/Option.types.ts

```diff
@@ -1,19 +1,47 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type OptionSlots = {
-  root: Slot<'div'>;
+  /* The root option slot, with role="option" */
+  root: NonNullable<Slot<'div'>>;
+
+  /* The check icon that is visible for selected options */
+  check: Slot<'span'>;
```

Comment:
Can we indicate somehow in the name that it's an icon? `checkIcon`, `checkmarkIcon`?